src/commons/TestUtils.ts

```diff
@@ -27,4 +30,24 @@ export namespace TestUtils {
   export function extractNumbersFromString(str: string): string {
     return str.replace(/[^0-9]/g, '');
   }
+
+  /**
+   * Return object of data by file path
+   *  path File located in wdio config file,
+   */
+  export function getDataFromFile(): object {
```

Comment:
using object will require to add ts-ignore in every usage. why not use generic types and pass it to the function?

Reply:
changed to generic type